client/components/combobox.less

```diff
@@ -23,16 +27,26 @@
           border-radius: 10px;
           border: 3px solid #ffffff;
         }
-
-        .item {
-            position:relative;
+		&.open {
+			display: unset;
+		}
+		> * {
+			all: unset;
+			display: block;
+			width: 100%;
+			position:relative;
             font-size: 11px;
             font-family: Open Sans;
             padding: 5px;
             cursor: default;
             margin: 0 3px;
+			box-sizing: border-box;
+			margin: 0;
             //border-bottom: 1px solid darkgray;
             &:hover {
+                background-color: rgb(223, 221, 221);
+			}
+			&:focus {
                 filter: brightness(120%);
                 background-color: rgb(163, 163, 163);
             }
```

Comment:
Overall, i'd say keep the colors in hex whenever possible, less characters, same legibility IMO

Reply:
Agreed, thanks for pointing that out.  I think it's a remnant from when i was trying to do tricky color manipulations here.  I went further and lightened the grays a bit more too.  This will be resolved when I push up my changes, so marking it now.